website/content/docs/job-specification/disk_throttle.mdx

```diff
@@ -0,0 +1,113 @@
+---
+layout: docs
+page_title: disk_throttles block in the job specification
+description: |-
+  Configure disk I/O throttling limits in the `disk_throttles` block of the Nomad job specification.
+---
+
+
+# `disk_throttle` block in the job specification
+
+
+<Placement groups={['job', 'group', 'task', 'resources', 'disk_throttle']} />
+
+The disk_throttle block is used to set limits on the disk I/O (Input/Output) a task can perform on a specific block device. 
```

Comment:
```suggestion
Use the `disk_throttle` block to set limits on the disk I/O (Input/Output) a task can perform on a specific block device. 
```

- Block name in code backticks
- [Use active voice ](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/top-12.md#write-in-active-voice)